lib/beaker/dsl.rb

```diff
@@ -79,5 +80,10 @@ module DSL
     include Beaker::DSL::Helpers
     include Beaker::DSL::InstallUtils
     include Beaker::DSL::Patterns
+    include Beaker::DSL::PE
+
+    def self.register(helper_module)
```

Comment:
I believe this came in as part of the work to support alternate test runners as part of Beaker.next (or some similar such name).

I don't know of a downside to the approach (though a question is whether it's in use by anything already or should have been rolled back when we rolled back an earlier PR). 

The downside to the current approach widely in use is that various modules directly open and reference the beaker class hierarchy and muck around with the internals. To eventually have sane libraries and to support things like alternate test runners we're likely to need to move away from that practice. Providing a registration hook is one way to do that.

I can talk more at some length about the bigger picture thinking (there's a mikado graph around here somewhere too) that motivates this.